src/js/actions/tweetsActions.js

```diff
@@ -1,37 +0,0 @@
-import axios from "axios";
-
-export function fetchTweets() {
-  return function(dispatch) {
```

Comment:
I think, this version of actions was better... You can only use array function here and then you'll have:
`return (dispatch) => { `